tutorials/mnist_tutorial_jsma.py

```diff
@@ -112,6 +112,14 @@ def mnist_tutorial_jsma(train_start=0, train_end=60000, test_start=0,
     grid_shape = (nb_classes, nb_classes, img_rows, img_cols, channels)
     grid_viz_data = np.zeros(grid_shape, dtype='f')
 
+    # by default, we have softmax after a CNN, remove it here
```

Comment:
The JSMA implementation would not change whether it is being fed probabilities or logits, it can handle both. That's why we were thinking of updating the documentation to make that more clear rather than modifying the tutorial (the few lines added to the tutorial in this PR would be removed). Does that make sense or do you prefer an alternative?